dev_apps/music-nga/open.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html class="theme-media">
+<head>
+  <meta charset="utf-8">
+  <title>Music NGA</title>
+  <meta name="description" content="Gaia Music powered by NGA">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
+  <link rel="stylesheet" href="/components/gaia-fonts/style.css">
+  <link rel="stylesheet" href="/components/gaia-icons/gaia-icons-embedded.css">
+  <link rel="stylesheet" href="/components/gaia-theme/gaia-theme.css">
+  <link rel="stylesheet" href="/css/app.css">
+  <script src="/components/gaia-header/dist/gaia-header.js"></script>
+  <script src="/elements/music-view-stack.js"></script>
+  <style>
```

Comment:
I think I'd rather have these CSS overrides in a separate file, especially since they're used for both activities. I'm not sure I like how we have to override the display of the tab bar either; maybe something where activities have a minimalist view stack implementation instead. I won't block review on that part, though.